src/main/java/org/jboss/threads/EnhancedQueueExecutor.java

```diff
@@ -630,6 +630,7 @@ public int getMaximumQueueSize() {
 
         /**
          * Set the maximum queue size.
+         * This has no impact when <pre>jboss.threads.eqe.unlimited-queue</pre> is set.
```

Comment:
This formats strangely.
```suggestion
         * This has no impact when {@code jboss.threads.eqe.unlimited-queue} is set.
```

Reply:
Good call, thanks!